### PR TITLE
fix(testing): disable collectCoverage by default

### PR DESCRIPTION
**Pull request by wesleygrimes**
Disabling collectCoverage by default as this causes issues with latest version of `ts-jest`

_[Please make sure you have read the submission guidelines before posting an PR](https://github.com/nrwl/nx/blob/master/CONTRIBUTING.md#submit-pr)_

> _Please make sure that your commit message follows our format._

> Example: `fix(nx): must begin with lowercase`

## Current Behavior (This is the behavior we have today, before the PR is merged)

`collectCoverage` is enabled by default, but causes issues with the line numbers and stack trace reported during test failures.

## Expected Behavior (This is the new behavior we can expect after the PR is merged)

`collectCoverage` will be disabled by default as a temporary workaround until `ts-jest` has been corrected.

## Issue

This is a temporary workaround for #1531 until issues are resolved with `ts-jest`

### CODE REVIEW

File: packages/schematics/migrations/update-7-7-0/update-7-7-0.spec.ts [hunk 1]
```diff
@@ -55,7 +55,6 @@ describe('Update 7.7.0', () => {
           },
           resolver: '@nrwl/builders/plugins/jest/resolver',
           moduleFileExtensions: ['ts', 'js', 'html'],
-          collectCoverage: true,
```
Comment (FrozenPandaz) [inline packages/schematics/migrations/update-7-7-0/update-7-7-0.spec.ts]: This is fine..

But we should also create an update for `8.3.0` which will remove the collectCoverage support...

Something like search for AST Nodes which is a property assignment for "collect coverage"

**Review by FrozenPandaz — CHANGES_REQUESTED** [review]

File: packages/schematics/migrations/update-7-7-0/update-7-7-0.spec.ts (hunk 1 above)
Comment (wesleygrimes) [reply packages/schematics/migrations/update-7-7-0/update-7-7-0.spec.ts]: Sounds good, I will get started on this.

Comment (wesleygrimes) [reply packages/schematics/migrations/update-7-7-0/update-7-7-0.spec.ts]: @FrozenPandaz I have added an 8.3.0 migration that will change the `collectCoverage` to `false` if it is found and currently set as `true`. I didn't see any migrations yet for 8.3.0, so let me know if there is a better place to do this.

Either way, should be ready for your review

File: packages/schematics/migrations/update-8-3-0/update-8-3-0.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import { Rule, SchematicContext, Tree } from '@angular-devkit/schematics';
+
+function updateJestConfig(host: Tree, context: SchematicContext) {
```
Comment (FrozenPandaz) [inline packages/schematics/migrations/update-8-3-0/update-8-3-0.ts]: Can you move this migration to `@nrwl/jest` please?

**Review by FrozenPandaz — CHANGES_REQUESTED** [review]

File: packages/schematics/migrations/update-8-3-0/update-8-3-0.ts (hunk 2 above)
Comment (wesleygrimes) [reply packages/schematics/migrations/update-8-3-0/update-8-3-0.ts]: Done with latest commit

**Review by FrozenPandaz — APPROVED** [review]

**Pull request merged**